lib/origami-build-tools-cli.js

```diff
@@ -96,6 +96,8 @@ if (tasks.isValid(argument)) {
 				taskResult
 					.on('error', reportTaskError)
 					.on('end', reportFinished);
+				// Make sure it reaches 'end'
+				taskResult.resume();
```

Comment:
The only one we don't understand is version, and we don't need to do anything, and if we don't understand the returned value we shouldn't do anything, right? I also think all tasks should return a promise or a stream and I would rather make version return a promise (though it's probably an overkill)